readme.md

```diff
@@ -36,6 +36,7 @@
     <li><a href="https://reactjs.org/">CSS</a></li>
     <li><a href="https://reactjs.org/">JavaScript</a></li>
     <li><a href="https://reactjs.org/">React</a></li>
+     <li><a href="https://redux.js.org/">Redux</a></li>
```

Comment:
- [ ] As suggested by the last reviewer, all related technologies should direct to the appropriate links. As you can see above, that's not the case here